docs/3.x.x/guides/deployment.md

```diff
@@ -423,6 +452,8 @@ const http = require('http');
 const crypto = require('crypto');
 const exec = require('child_process').exec;
 
+const PM2_CMD = 'cd ~ && pm2 startOrRestart ecosystem.config.js';
```

Comment:
Here you might give two options: 

`pm2 restart strapi` or using the `ecosystem` in case the user followed the ecosystem step (which is optionnal)